allennlp/service/predictors/crf_tagger.py

```diff
@@ -14,15 +14,33 @@ class CrfTaggerPredictor(Predictor):
     """
     def __init__(self, model: Model, dataset_reader: DatasetReader) -> None:
```

Comment:
Couldn't you use this `Predictor` with a trained `SimpleTagger` also?  Why is it called `CrfTaggerPredictor`?

Reply:
I think you're right, but there are some complications around the way we're using predictor names as the source of truth in our front-end.

I think we need to come up with different abstractions than the `DEFAULT_MODELS` and `DEFAULT_PREDICTORS` dicts that we have in `commands/__init__.py`. I have always felt that's not the right way to do it, let me noodle on it a bit.